src/content/developers/docs/networks/index.md

```diff
@@ -68,7 +66,7 @@ A proof-of-authority testnet that works across clients; an ideal testnet for app
 
 _Note, [the Ropsten testnet is deprecated](https://github.com/ethereum/pm/issues/460) and will no longer receive protocol upgrades. Please consider migrating your applications to Sepolia or Goerli._
```

Comment:
Ropsten IS deprecated? Or will be deprecated.

Reply:
already deprecated (i.e. no active maintenace), will be shut down in Q4

Reply:
Ah, the language isn't clear here then. We're using deprecated to mean both: 

- no longer maintained
- no longer accessible